eva_cttv_pipeline/evidence_string_generation/clinvar.py

```diff
@@ -18,8 +17,17 @@ class ClinvarRecord(UserDict):
 
     def __init__(self, cellbase_dict):
         UserDict.__init__(self, cellbase_dict)
-        self.measures = [ClinvarRecordMeasure(measure_dict, self)
-                         for measure_dict in self.data['referenceClinVarAssertion']["measureSet"]["measure"]]
+        if "measureSet" in self.data['referenceClinVarAssertion']:
+            measure_list = self.data['referenceClinVarAssertion']["measureSet"]["measure"]
+        elif "measureSet" in self.data['referenceClinVarAssertion']["genotypeSet"]:
+            measure_list = []
+            for measure_set in self.data['referenceClinVarAssertion']["genotypeSet"]["measureSet"]:
+                for measure in measure_set["measure"]:
+                    measure_list.append(measure)
```

Comment:
so, if `measureSet` is present in referenceClinVarAssertion, it has only a simple list under the key `measure`, but if `measureSet` is under genotypeSet, `measureSet` has a list of objects, each one with the key `measure` that is a list?